### PR TITLE
Chinese version translated/翻译了中文版本

### PR DESCRIPTION
**Pull request by zhiquanchi**
Chinese version/中文版本

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by diemol** [comment]
Can you please sign the CLA?

File: website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md [hunk 1]
```diff
@@ -1,65 +1,54 @@
 ---
-title: "RemoteWebDriver BiDirectional API"
-linkTitle: "RemoteWebDriver BiDi API"
+title: "远程网络驱动 双向 API（RemoteWebDriver BiDirectional API）"
+linkTitle: "远程网络驱动 BiDi API"
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md]: Can you keep the WebDriver as a Proper Noun ？

Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md]: Can you keep the WebDriver as a Proper Noun ？

File: website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md [hunk 2]
```diff
@@ -1,65 +1,54 @@
...
 {{< tab header="Java" >}}
     {{< gh-codeblock path="examples/java/src/test/java/dev/selenium/augmenter/CdpRemoteWebDriverTest.java#L111-133" >}}
 {{< /tab >}}
 {{< /tabpane >}}
 
-## Listen to `console.log` events
+## 监听控制台日志事件 （Listen to `console.log` events）
 
-Listen to the `console.log` events and register callbacks to process the event.
+侦听 console. log 事件并注册回调以处理该事件。
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md]: please keep original format as the command and do not break the method call with external space

Comment (alaahong) [reply website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md]: why the title is “监听” but change to “侦听” in content？

File: website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md [hunk 3]
```diff
@@ -1,65 +1,54 @@
...
+## JS异常处理 （Actions causing JS exceptions）
 
 {{< tabpane disableCodeBlock=true >}}
 {{< tab header="Java" >}}
     {{< gh-codeblock path="examples/java/src/test/java/dev/selenium/augmenter/CdpRemoteWebDriverTest.java#L179-185" >}}
 {{< /tab >}}
 {{< /tabpane >}}
 
-## Network Interception
+## 监听网络 （Network Interception）
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md]: It's not monitor only.

**Comment by alaahong** [comment]
Thanks for your first contribution. Just leave some comments, please feel free to discuss.
Generally, it's the "Improved translation", you should keep the wording under same translation in one post, and meanwhile, the conversion is better to avoid long sentence in one line. At last, keep original format(e.g. `command`) and punctuation(e.g. half-width symbol)  is friend to validate the layout.

File: website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md (hunk 3 above)
Comment (zhiquanchi) [reply website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md]: I will change then translation.I think should use '截网侦察'

File: website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md (hunk 1 above)
Comment (zhiquanchi) [reply website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md]: Yes,i can.
I will to change it.

File: website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md [hunk 4]
```diff
@@ -1,65 +1,54 @@
...
-   Page being translated from
-   English to Chinese. Do you speak Chinese? Help us to translate
-   it by sending us pull requests!
-</p>
-{{% /pageinfo %}}
 
-The following examples demonstrate how to leverage BiDi APIs with [Remote WebDriver](/documentation/webdriver/remote_webdriver/).
 
-## Register Basic Auth
+下面的例子显示如何使用 [Remote WebDriver](/documentation/webdriver/remote_webdriver/).
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md:9]: Where is BiDi APIs ?

File: website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md [hunk 5]
```diff
@@ -1,65 +1,54 @@
...
 {{< tab header="Java" >}}
     {{< gh-codeblock path="examples/java/src/test/java/dev/selenium/augmenter/CdpRemoteWebDriverTest.java#L111-133" >}}
 {{< /tab >}}
 {{< /tabpane >}}
 
-## Listen to `console.log` events
+## 监听控制台日志事件 （Listen to `console.log` events）
 
-Listen to the `console.log` events and register callbacks to process the event.
+监听 console. log 事件并注册回调以处理该事件。
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md:33]: There is a code , please keep format as previous mentioned

**Review by alaahong — CHANGES_REQUESTED** [review]
Almost are fine, only need aware the missed parts as mentioned.

**Review by alaahong — CHANGES_REQUESTED** [review]
Hi, will you do the post commit to avoid impact original description and code segment?

**Pull request closed without merging**